demo/resources.hpp

```diff
@@ -117,7 +119,11 @@ namespace resources
 		mirror_internal->SetAlbedo(white);
 		mirror_internal->SetNormal(flat_normal);
 		mirror_internal->SetRoughness(black);
+		mirror_internal->UseRoughnessTexture(false);
+		mirror_internal->SetConstantRoughness(0.0f);
```

Comment:
This causes the difference between deferred/hybrid & fullraytracing. That constant value doesn't work properly.